<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnknownVersion</class>
 <widget class="QDialog" name="UnknownVersion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Unknown Zandronum Version</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="m_description">
     <property name="text">
      <string>The demo &lt;DEMO&gt; has been recorded with unknown version &lt;VERSION&gt;. You can add this to the list of known Zandronum versions and specify a binary for it here.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Zandronum binary path:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="m_binaryPath"/>
       </item>
       <item>
        <widget class="QPushButton" name="m_findBinary">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="m_addVersion">
     <property name="text">
      <string>Add Zandronum version and launch the demo</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="m_cancel">
     <property name="text">
      <string>Cancel and exit</string>
     </property>
     <property name="description">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_cancel</sender>
   <signal>clicked()</signal>
   <receiver>UnknownVersion</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
